build-support/build-opts.js

```diff
@@ -0,0 +1,66 @@
+const execSync = require('sync-exec')
+const { env } = process
+const production = !!env.production
+const src = './src'
+const dest = env.dest ? env.dest : './dist'
+const noGlobals = !!env.noGlobals // don't generate globals bundle
+const noMap = !!env.noMap // don't generate .map files
+const codap = !!env.codap // include CODAP-specific modifications
+const assetsSrc = codap ? src + '/assets/img/*.*' : src + '/assets/**/*.*'
+const assetsDst = codap ? dest + '/img/' : dest
+
+const commit = execSync('git log -1 --pretty=format:"%H"').stdout
+const date = new Date()
+
+// CODAP builds append '.ignore' to js filenames to avoid CODAP compilation
+const codapOutputFileName = (webpackChunk) => {
+  return webpackChunk.chunk.name.match(/\.js$/) ? '[name].ignore': '[name]';
```

Comment:
I think we should use `.codap` instead of `.ignore` for clarity. It's a trivial change to make on the CODAP side and is more self-explanatory from the CFM perspective.